src/libraries/Microsoft.PowerFx.Core/Microsoft.PowerFx.Core.csproj

```diff
@@ -21,6 +21,7 @@
 		<Description>The core library for all Microsoft Power Fx Compiler targets</Description>
 		<Summary>The core library for all Microsoft Power Fx Compiler targets</Summary>
 		<GeneratePackageOnBuild>True</GeneratePackageOnBuild>
+		<GenerateNeutralResourcesLanguageAttribute>false</GenerateNeutralResourcesLanguageAttribute>
```

Comment:
This needs to be disabled so that the en-us resources can also be loaded from a satellite.